dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/analytics/outlier/OutliersDetection2AutoTest.java

```diff
@@ -340,11 +337,11 @@ public void queryOutliertest8() throws JSONException {
     validateHeader(response, 2, "zscore", "Z-score", "NUMBER", "java.lang.Double", false, false);
 
     // Assert rows.
-    validateRow(response, 0, List.of("l6byfWFUGaP", "Yellow Fever doses given", "3.7"));
-    validateRow(response, 1, List.of("s46m5MS0hxu", "BCG doses given", "3.1"));
+    validateRow(response, List.of("l6byfWFUGaP", "Yellow Fever doses given", "3.66"));
```

Comment:
Is there a reason why you removed the indexes? If we have the `orderBy=zscore`, we should assert that the order is correct. I would keep the indexes. If the `orderBy` is not working it should be fixed...

Reply:
done